src/bindgen/ir/function.rs

```diff
@@ -183,6 +189,51 @@ impl Function {
                 reserved::escape(name);
             }
         }
+
+        // Save the array length of the pointer arguments which need to use
+        // the C-array notation
+        if let Some(tuples) = self.annotations.list("ptrs-as-arrays") {
+            let mut ptrs_as_arrays: HashMap<String, String> = HashMap::new();
+            let mut unnamed_ptrs: Vec<String> = Vec::new();
+            for str_tuple in tuples {
+                let parts: Vec<&str> = str_tuple[1..str_tuple.len() - 1]
+                    .split(';')
+                    .map(|x| x.trim())
+                    .collect();
+                if parts.len() != 2 {
+                    warn!(
+                        "{:?} does not follow the correct syntax, so the annotation is being ignored",
+                        parts
+                    );
+                    continue;
+                }
+                if parts[0] == "_" {
+                    unnamed_ptrs.push(parts[1].to_string());
```

Comment:
It seems weird that the unnamed argument order is positional and gets removed even if the argument is not a pointer... I'd probably just not handle unnamed arguments altogether, just not crashing is fine :)

Reply:
How would you treat them? Since they don't have a proper name, I thought it would have been better to use a positional approach. I think the unnamed arguments should be considered, perhaps someone could be interested in this kind of conversion `_: *const T --> const T[array_length]` in a code

Reply:
I can add a check in order to push only arguments that are pointers

Reply:
Well, they can name the arguments in that case, IMO.

Reply:
I think it could be a constraint for some developers, that's why I have discarded that approach initially, but effectively you're right, loss boilerplate for us :smile:

Reply:
People that care about describing the intent of an argument would probably also name it, so I do not see it as something somebody would miss.